tests/drivers/flash/flash_rpc/Kconfig.sysbuild

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "${ZEPHYR_BASE}/share/sysbuild/Kconfig"
+
+config REMOTE_BOARD
+	string "The board used for remote target"
```

Comment:
Should be using a qualifier not a board name, and it can be defaulted in here e.g.
```
default "cpunet" if SOC_SERIES_NORDIC_NRF53X
```